civiwiki/civiwiki/urls.py

```diff
@@ -25,7 +25,8 @@
     url(r'^api/', include(urls_write)),
     url(r'^api/', include(urls_read)),
     url(r'^api/', include(urls_auth)),
-    url(r'^',include(urls_templates)),
+    url(r'^/',include(urls_templates)),
+    url(r'^$',include(urls_templates)),
```

Comment:
Regex my friend. This is matching the empty string but not anything else, without that end tag it matches everything.